core/test/base/exception_helpers.cpp

```diff
@@ -40,47 +40,47 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 namespace {
 
 
-void not_implemented_func() NOT_IMPLEMENTED;
+void not_implemented_func() GKO_NOT_IMPLEMENTED;
 
 TEST(NotImplemented, ThrowsWhenUsed)
 {
     ASSERT_THROW(not_implemented_func(), gko::NotImplemented);
```

Comment:
I understand correct that these are macros from the C++ standard?

Reply:
Yes. Well, technically they come from googletest: https://github.com/google/googletest/blob/50f1a77955bde27cf49173c4133688d61f201e85/googletest/docs/advanced.md#exception-assertions